src/main/java/org/opentripplanner/updater/alerts/GtfsRealtimeAlertsUpdater.java

```diff
@@ -81,6 +90,9 @@ protected void configurePolling(Graph graph, JsonNode config) throws Exception {
         if (config.path("fuzzyTripMatching").asBoolean(false)) {
             this.fuzzyTripMatcher = new GtfsRealtimeFuzzyTripMatcher(graph.index);
         }
+        this.connectionTimeout = config.path("connectionTimeoutMills").asInt();
+        this.connectionRequestTimeout = config.path("connectionRequestTimeoutMills").asInt();
+        this.socketTimeout = config.path("socketTimeoutMills").asInt();
```

Comment:
If the JsonNode does not have the "connectionTimeoutMills" field is there a likelihood of an NPE?

Reply:
No, there is no risk of an NPE. From the Javadoc on path(): "This method is similar to get(String), except that instead of returning null if no such value exists (due to this node not being an object, or object not having value for the specified field), a "missing node" (node that returns true for isMissingNode()) will be returned. This allows for convenient and safe chained access via path calls."